pkgs/tools/package-management/rpm/default.nix

```
@@ -28,14 +29,16 @@ stdenv.mkDerivation rec {
     "--with-external-db"
     "--with-lua"
     "--enable-python"
+    "--enable-ndb"
+    "--enable-sqlite"
     "--localstatedir=/var"
     "--sharedstatedir=/com"
   ];
 
-  postPatch = ''
-    # For Python3, the original expression evaluates as 'python3.4' but we want 'python3.4m' here
-    substituteInPlace configure.ac --replace 'python''${PYTHON_VERSION}' ${python.executable}
+  # Small fixes for ndb on darwin
+  patches = lib.optional stdenv.isDarwin ./darwin.patch;
```

Since the patch is a noop on other platforms I rather want to enable it on other platforms to avoid bitrot of the patch. Has an attempt made to submit this patch upstream?

```suggestion
  patches = [ ./darwin.patch ];
```

---

It has!
https://github.com/rpm-software-management/rpm/pull/1465

I wasn't quite sure if it feels a bit weird to have darwin specific patch that still runs on linux (even though it's a noop).

Happy to change it though :)

---

We quite often have linux testers and no good macOS CI right now, so updates might break the patch going unnoticed.